app/src/main/java/com/poklad/androidifystore/utils/Resource.kt

```diff
@@ -3,5 +3,7 @@ package com.poklad.androidifystore.utils
 sealed class Resource() {
     class Success<T>(data: T) : Resource()
     class Error(val throwable: Throwable?) : Resource()
+    //todo а может нужні данные чтобы отображать что-то даже если не дозвонились до сервера
+    //если во вьюмодел, то норм, если будет в Usecasr  то нужно данные передавать вроде.
```

Comment:
this is a specific case, so in that case throw exception smth like `TimeoutException` and in viewmodel (or even better in baseViewModel handle it and pass to an appropriate liveData/flow)